app/src/main/java/io/lbry/browser/tasks/CommentListTask.java

```diff
@@ -0,0 +1,73 @@
+package io.lbry.browser.tasks;
+
+import android.os.AsyncTask;
+import android.view.View;
+import android.widget.ProgressBar;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import io.lbry.browser.model.Comment;
+import io.lbry.browser.utils.Helper;
+import io.lbry.browser.utils.Lbry;
+
+public class CommentListTask extends AsyncTask<Void, Void, List<Comment>> {
+    private String claim;
+    private ProgressBar progressBar;
+    private CommentListHandler handler;
+    private Exception error;
+
+    public CommentListTask(String claim, ProgressBar progressBar, CommentListHandler handler) {
```

Comment:
Where are new comments added? I'm a little worried about comments getting added while users are scrolling.

Reply:
I added the handling but I didn't implement it (I don't _think_ you requested I implement and I'm not sure how to do that because of the above question so I just added handling). 9e6acd9

Reply:
> Where are new comments added? I'm a little worried about comments getting added while users are scrolling.

Usually, you could use a flag to check if comments are loading and then simply return instead of sending another request to fetch the next page. I'll accept this and make some tweaks.